src/clr-angular/button/_variables.button.scss

```diff
@@ -5,27 +5,52 @@
 // General
 // Usage: ./_buttons.clarity.scss
 // Usage: ../popover/tooltip/_tooltips.clarity.scss
-$clr-button-vertical-margin: 0.25rem !default;
+$clr-button-vertical-margin: 0.25rem !default; // DEPRECATED
```

Comment:
Does it make sense to annotate the DEPRECATED's instead of commenting?

```/// @deprecated Deprecation related message```

Reply:
Does that do anything? Or is it just for consistency? I couldn't find any use of `@deprecated` in our codebase.